src/__tests__/fireEvent.test.tsx

```diff
@@ -475,3 +475,53 @@ test('has only onMove', () => {
   });
   expect(handleDrag).toHaveBeenCalled();
 });
+
+// Those events ideally should be triggered through `fireEvent.scroll`, but they are handled at the
+// native level, so we need to support manually triggering them
+describe('native events', () => {
```

Comment:
```suggestion
describe('native event support', () => {
```

Reply:
I wonder if we should actually use `host component event support` instead, as by design we do not support "native" events.